src/EditorFeatures/Test2/IntelliSense/SignatureHelpControllerTests.vb

```diff
@@ -249,6 +250,7 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
                          </Workspace>)
                      Return workspace.CurrentSolution.GetDocument(workspace.Documents.Single().Id)
```

Comment:
➡️ In round one I opted for that approach, but in the end I decided to go for the smaller change. I can file a follow-up issue but it didn't seem particularly important since this is test code.